backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -526,10 +521,11 @@ public async System.Threading.Tasks.Task<Project> UpdateAsync(Project project)
             var responses = project.GetResponseChanges(project);
 
             // If the note was changed generate a notification for it.
-            var noteChanged = originalProject.PublicNote != project.PublicNote;
+            var noteChanged = !String.IsNullOrWhiteSpace(project.GetNoteText(NoteTypes.Public)) && originalProject.GetNoteText(NoteTypes.Public) != project.GetNoteText(NoteTypes.Public);
```

Comment:
so deleting a note should not generate a notification?

Reply:
Correct.  The current template includes the new note.  Which would be empty and cause more confusion.  I didn't want to increase scope to update the template to give a different message.  Additionally if they use this as a tool to communicate with the agency in some fashion, it doesn't really make sense to send an email when there is no note.